src/_pytest/config/findpaths.py

```diff
@@ -107,8 +108,6 @@ def determine_setup(inifile, args, rootdir_cmd_arg=None, config=None):
             try:
                 inicfg = iniconfig[section]
                 if is_cfg_file and section == "pytest" and config is not None:
-                    from _pytest.deprecated import CFG_PYTEST_SECTION
-
                     fail(
```

Comment:
would a `UsageError` fit here?

Reply:
We could, but then we can't pass `pytrace=False`, which I think it is better as we then hide the pytest internals